swan-lake/learn-the-language/distinctive-language-features/data.md

```diff
@@ -884,13 +884,17 @@ The ``==`` operator does a deep equals comparison.
 You can loop through the ``xml`` elements in a ``foreach`` statement.
 
 ```ballerina
+import ballerina/io;
+
 xml x4 = xml `<name>Sherlock Holmes</name><details>
```

Comment:
IIRC, with these guides, the snippets don't have to compile on its own, since we are explaining step by step.

If we are doing the changes here, let's make this also a local variable.